os/net/mac/tsch/tsch-slot-operation.c

```diff
@@ -167,6 +167,13 @@ static struct tsch_link *backup_link = NULL;
 static struct tsch_packet *current_packet = NULL;
 static struct tsch_neighbor *current_neighbor = NULL;
 
+/* Indicates whether an extra link is needed to handle the current burst */
+static int burst_link_scheduled = 0;
+/* Counts the length of the current burst */
+int tsch_current_burst_count = 0;
+/* The physical channel of the current burst */
+static uint16_t burst_tsch_current_channel = 0;
```

Comment:
I don't immediately see the need for this new variable. Why is `tsch_current_channel` not enough? While in burst mode, that variable should not be updated.

Reply:
agree